### PR TITLE
Multiple Genome Catalogues

### PR DESCRIPTION
**Pull request by SandyRogers**
This PR:
- removes top-level "Genomes" page and nav bar item
- adds "Genome catalogues" as a tab of the Browse page
- adds listing table of genome catalogues

TODO once merged:
- remove branch hack from test.yml (due to bug on a GitHub Actions workflow we use)

### CODE REVIEW

File: src/js/modules/genome.js [hunk 1]
```diff
@@ -287,6 +287,21 @@ let DownloadsView = Backbone.View.extend({
...
 
+let GenomeCataloguesView = Backbone.View.extend({
+    template: _.template($('#cataloguesTmpl').html()),
+    el: '#catalogues-section',
+    init() {
+        const that = this;
+        return this.collection.fetch({
+            success() {
+                const data = {catalogues: that.collection.toJSON()};
+                console.log(data);
```
Comment (mberacochea) [inline src/js/modules/genome.js:298]: wild console.log statement

**Review by mberacochea — COMMENTED** [review]
Looks good.. Could we add some unit tests for this (or a JIRA task to write them later)?.

**Comment by SandyRogers** [comment]
> Looks good.. Could we add some unit tests for this (or a JIRA task to write them later)?.

Thanks. There is existing coverage from unit tests for Browse (because genomes moved there) and Genome detail (existing test). There is a previously-disabled test for genomes listing which I haven't tackled fixing yet... Will add a task.

File: src/genome.html [hunk 2]
```diff
@@ -56,8 +56,11 @@ <h2>Genome <% print(accession) %></h2>
                     <li class="tabs-title"><a href="#kegg-module-analysis">KEGG module analysis</a></li>
                     <li class="tabs-title"><a href="#downloads">Downloads</a></li>
                 </ul>
+                </ul>
```
Comment (gustavo-salazar) [inline src/genome.html:59]:
```suggestion
```

File: src/genome.html [hunk 3]
```diff
@@ -91,6 +94,32 @@ <h2>Genome <% print(accession) %></h2>
             <p>A full copy of the raw analysis data is available in the <a id="downloads-link-bottom" href="javascript:void(0);">downloads tab</a>.</p>
         </div>
 
+        <script id="cataloguesTmpl" type="text/template">
```
Comment (gustavo-salazar) [inline src/genome.html:96]: Sorry a bit confused... where is this used? I mean I see it been used in `genome.js`, but I'm still haven't found it while browsing in the website.

Comment (SandyRogers) [reply src/genome.html:96]: Great catch. This was relevant to a previous iteration where we showed links to all the catalogues in which a genome was present. This is now one-to-one so no longer used. Deleting.

**Review by gustavo-salazar — APPROVED** [review]
Cool. I'm happy with the changes. I'll start the integration of sourmash into it!

**Pull request merged**